packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-editors-list/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,6 @@
 {
   "pci_notebook_add_editors_title": "Choix du live-code editor",
   "pci_notebook_add_editors_editor_vscode": "Visual Studio Code",
-  "pci_notebook_add_editors_editor_jupyterlab": "Jupyter"
+  "pci_notebook_add_editors_editor_jupyterlab": "JupyterLab",
+  "pci_notebook_add_editors_editor_jupyterlabcollaborative": "JupyterLab + Real Time Collaboration"
```

Comment:
**question:** Do we expect actual content to be translated into other languages? If no, then could we consider moving it to a constant?

Reply:
hello Antoine, indeed for now we will use the same wording for all languages
For the constant, we would prefer not.

- The 2 others naming are not using constant (homogeneity)
- Il will require to create new files
- we are not future proof if we decide to translate it later

Thank you !